lbuild/config.py

```diff
@@ -222,6 +223,13 @@ def from_file(configfile, parent=None):
             rfilename = ConfigNode._rel_path(repopath, configpath)
             config._repositories.append(rfilename)
 
+        # Load glob repositories (optional)
+        for glob_node in xmltree.iterfind("repositories/glob/path"):
+            globPath = glob_node.text.format(cache=config._cachefolder)
+            for f in glob.glob(globPath):
```

Comment:
rglob? `ext/**/repo.lb` could be useful.

Reply:
Dunno, I didn't want to go "that far" by doing it recursive as well :) should probably be configurable with `recursive=False` by default.

Reply:
Ah, you're right, glob alone already supports `**`, and you can get the same behavior as rglob by prefixing the path with `**/`.